src/Components/ColorPicker/ColorPicker.scss

```diff
@@ -94,22 +94,3 @@ $color-picker--margin: 0;
     padding-left: $cf-marg-d - $cf-marg-a;
   }
 }
-
-.cf-color-picker--selected {
-  pointer-events: none;
-  z-index: 2;
-  position: absolute;
-  top: 50%;
-  left: $cf-marg-c;
-  transform: translate(-50%, -50%);
-  width: 18px;
-  height: 18px;
-  border-radius: 50%;
-  border: $cf-border solid $g5-pepper;
-  transition: background-color $cf-transition-default,
-    border-color $cf-transition-default;
-
-  .cf-input:hover + & {
-    border-color: $g7-graphite;
-  }
-}
```

Comment:
does this removal affect the existing color picker?

Reply:
This removal doesn't affect the existing color picker even though this portion of the code is moved to a separate file "src/Components/ColorPicker/ColorPreview.scss." I also double checked by looking into Google chrome dev tool. When the `cf-input` element is on hover, the border-color for color preview does change.